plugin/adsk/scripts/mayaUsd_createStageFromFile.mel

```diff
@@ -273,5 +273,9 @@ global proc mayaUsd_createStageFromFile()
 
 global proc string mayaUsd_createStageFromFilePath(string $usdFile)
 {
+    // This proc is used by the unit tests (which run in standalone mode).
+    // In that case we need to make sure the optionVars are initialized as
+    // they are queried in doCreateStage().
+    setOptionVars(false);
```

Comment:
So what happens is on line 141 it would do:
`string $ppath = `optionVar -q stageFromFile_primPath`;`
If we are running in standalone (say from a unit test) the options vars would not have been init, and in Maya when you query an option var that doesn't exist it returns 0. The MEL code above converts that to a string and you get "0". That would then we set as the primPath on the stage. In Dan's new code this is not a valid primPath so the compute would bail early. With this fix the option vars are init and it will set "" as the primpath which converts to the root prim "/" and the compute works.